analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -350,7 +350,7 @@ class SpdxDocumentFile(
         val externalDocumentReference = doc.externalDocumentRefs.find {
             it.externalDocumentId == documentRef
         } ?: throw IllegalArgumentException(
-            "No single package or externalDocumentRef with ID '$identifier' found."
+            "ID '$identifier' could neither be resolved as a package nor as an externalDocumentRef."
```

Comment:
"resolved to" instead of "resolved as".

Reply:
I was using "as" intentionally, but I'm also fine with "to". Will change in both places.

Reply:
Or did you really only want to get "as" changed to "to" in the first occurrence?

Reply:
I meant both occurrences of "as", I'm not a native speaker but "resolve as" just sounds wrong to me.

Reply:
Anyway, if you want to keep it as is that's fine with me.

Reply:
Changed in both places.